src/main/java/gyro/aws/secretsmanager/SecretResource.java

```diff
@@ -0,0 +1,474 @@
+    }
+
+    public void setTags(Map<String, String> tags) {
+        this.tags = tags;
+    }
+
+    /**
+     * The Amazon Resource Name (ARN) of the secret. This is unique.
+     */
+    @Id
+    @Output
+    public String getArn() {
+        return arn;
+    }
+
+    public void setArn(String arn) {
+        this.arn = arn;
+    }
+
+    /**
+     * This value exists if the secret is scheduled for deletion and specifies the date.
+     */
+    @Output
+    public String getDeletedDate() {
+        return deletedDate;
+    }
+
+    public void setDeletedDate(String deletedDate) {
+        this.deletedDate = deletedDate;
+    }
+
+    /**
+     * Specifies that the secret is to be deleted without any recovery window. Cannot use both this parameter and the
+     * RecoveryWindowInDays parameter in the same API call. See `Force Delete Without Recovery Info
+     * <https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_DeleteSecret.html#SecretsManager-DeleteSecret-request-ForceDeleteWithoutRecovery/>`_.
+     */
+    @ConflictsWith("recovery-window-in-days")
+    public Boolean getForceDeleteWithoutRecovery() {
+        return forceDeleteWithoutRecovery;
+    }
+
+    public void setForceDeleteWithoutRecovery(Boolean forceDeleteWithoutRecovery) {
+        this.forceDeleteWithoutRecovery = forceDeleteWithoutRecovery;
+    }
+
+    /**
+     * The last date that this secret was accessed.
+     */
+    @Output
+    public String getLastAccessedDate() {
+        return lastAccessedDate;
+    }
+
+    public void setLastAccessedDate(String lastAccessedDate) {
+        this.lastAccessedDate = lastAccessedDate;
+    }
+
+    /**
+     * The last date and time that this secret was modified in any way.
+     */
+    @Output
+    public String getLastChangedDate() {
+        return lastChangedDate;
+    }
+
+    public void setLastChangedDate(String lastChangedDate) {
+        this.lastChangedDate = lastChangedDate;
+    }
+
+    /**
+     * The most recent date and time that the Secrets Manager rotation process was successfully completed.
+     */
+    @Output
+    public String getLastRotatedDate() {
+        return lastRotatedDate;
+    }
+
+    public void setLastRotatedDate(String lastRotatedDate) {
+        this.lastRotatedDate = lastRotatedDate;
+    }
+
+    /**
+     * The name of the secret. (Required)
+     */
+    @Required
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * The name of the owning service.
+     */
+    @Output
+    public String getOwningService() {
+        return owningService;
+    }
+
+    public void setOwningService(String owningService) {
+        this.owningService = owningService;
+    }
+
+    /**
+     * Specifies the number of days that Secrets Manager waits before it can delete the secret. Cannot use both this
+     * parameter and the ForceDeleteWithoutRecovery parameter in the same API call.
```

Comment:
We will handle this differently, so just use the description
```suggestion
     * Specifies the number of days that Secrets Manager waits before it can delete the secret. 
```